woocommerce/woocommerce-ios
# Read-only Refunds, part 1

This PR takes the first steps to displaying refunds UI in the Order Details screen. Please excuse the large PR...I realized half way through my additions that OrderDetailsDataSource was huge and overwhelming, and refactored it.

## Changes
- Discovered more Yosemite work that I previously missed. Added `OrderRefundsCondensed` to Yosemite.
- Originally, I publicly exposed the footerLabel in `TwoColumnHeadlineFooterTableViewCell`. But then I realized it violates the open / closed principle.
- Created a new tableviewcell, `TwoColumnHeadlineAttributedFooterTableViewCell`, to remedy this
- ~Refactored `OrderDetailsDataSource`. Broke it into 4 files: main, Constants, Cells, and Sections. If there are parts that need moved (so it makes sense), please let me know!~
- Feature flagged Refunds. 
- The Refund cell(s) appear on Order Details for an order containing 1 or more refunds.
- The Net cell appears on Order Details when a refund is present. This cell has not been fully configured, all it has is "Net" for the title.

*Known Issues:*
- Currency formatting for Refund & Net cell full configuration to come in another PR
- Net footnote text to be removed in another PR
- View details > Refund Details screen to come in another PR

## To test
1. Find an order in your store
2. Create a partial refund on the desktop web interface
3. Navigate to the correct order number
4. Expected result: Refund cell(s) display. The Net amount cell appears (but doesn't have any real info yet).

iPhone 8:
<kbd>
<img src="https://user-images.githubusercontent.com/1062444/69279353-c7f01500-0ba9-11ea-9470-0d24ecae569f.png" width="350" />
</kbd>

Update release notes:
